.github/workflows/build.yaml

```diff
@@ -0,0 +1,68 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: build
+        run: make build
+      - name: get tools
+        run: ./contrib/get_tools.sh
+      - name: lint chart
+        run: ./contrib/lint_chart.sh
+      - name: create cluster
+        run: ./contrib/create_cluster.sh
+      - name: deploy deploy
+        run: OPERATOR=0 ./contrib/deploy.sh
+      - name: run function
+        run: ./contrib/run_function.sh
+      - name: stop dev cluster
+        run: ./contrib/stop_dev.sh
+      - name: Sleep for 10 seconds
+        uses: jakejarvis/wait-action@master
+        with:
+          time: '10s'
+      - name: create cluster
+        run: ./contrib/create_cluster.sh
+      - name: deploy function
+        run: OPERATOR=1 ./contrib/deploy.sh
+      - name: run function
+        run: OPERATOR=1 ./contrib/run_function.sh
+      - name: stop dev cluster
+        run: ./contrib/stop_dev.sh
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Docker Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: ghcr.io
+      - name: Push containers
+        uses: docker/build-push-action@v2
+        with:
+          outputs: "type=registry,push=true"
```

Comment:
I am conflicted about filling up the storage with images upon each commit into HEAD. Let's remove this for the time being.

Reply:
If people want to try out solutions before their release via a tag, this might be beneficial. We could have a separate flow for pushing in to master for just this specifically, for the time being I'll remove the publishing of the images tho, as requested.